doc/source/_includes/rllib/new_api_stack.rst

```diff
@@ -8,4 +8,4 @@
     support the "new API stack" and continue to run by default with the old APIs.
     You can continue to use the existing custom (old stack) classes.
 
-    `See </rllib/package_ref/rllib-new-api-stack.html>`__ for more details on how to use the new API stack.
\ No newline at end of file
+    `See </rllib/rllib-new-api-stack.html>`__ for more details on how to use the new API stack.
```

Comment:
```suggestion
    See `RLlib’s New API Stack <../../rllib/rllib-new-api-stack.html>`__ for more details on how to use the new API stack.
```

Reply:
The link is not resolving for me in the CI build. I've suggested a fix, but I didn't test it out. Please verify that it's correct.